src/tarantool-client/Converters/EvalPacketConverter.cs

```diff
@@ -222,7 +222,7 @@ public void Write(EvalPacket<T1, T2, T3, T4, T5, T6, T7, TRest> value, IMsgPackW
             stringConverter.Write(value.Expression, writer, context);
 
             keyConverter.Write(Key.Tuple, writer, context);
-            writer.WriteArrayHeader(1);
+            writer.WriteArrayHeader(8);
```

Comment:
Yes, you are right.
Key.Tuple should not be written in TupleConverter, but array header should.